src/webapp/azext_webapp/create_util.py

```diff
@@ -71,6 +73,9 @@ def get_runtime_version_details(file_path, lang_name):
     elif lang_name.lower() == STATIC_RUNTIME_NAME:
         version_detected = "-"
         version_to_create = "-"
+    elif lang_name.lower() == PYTHON_RUNTIME_NAME:
+        version_detected = "-"
```

Comment:
Need to follow up with PM on available version for Linux & detecting this reliably

Reply:
Follow-up with PM: Only a single version is supported right now so this is good